### PR TITLE
Make sure drone checks acutally work the right way around

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Make sure drone checks acutally work the right way around

### CODE REVIEW

**Comment by zero-24** [comment]
@joomla/cms-maintainers @joomla/cms-maintainers-admin @joomla/ci-admins  please take a look here and hit merge. Thanks.

File: libraries/vendor/phpunit/dbunit/composer/bin/dbunit [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env php
```
Comment (wilsonge) [inline libraries/vendor/phpunit/dbunit/composer/bin/dbunit]: Don't think this is supposed to be here

**Comment by zero-24** [comment]
just removed that file. I'm not sure where it was coming from.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**